packages/web-components/fast-components/src/listbox-option/listbox-option.styles.ts

```diff
@@ -43,113 +39,93 @@ export const optionStyles: FoundationElementTemplate<
-    }
-
-    :host(:hover) {
-        background: ${accentFillHover};
-        color: ${foregroundOnAccentHover};
-    }
-
-    :host(:active) {
-        background: ${accentFillActive};
-        color: ${foregroundOnAccentActive};
-    }
-
-    :host(:not([aria-selected="true"]):hover),
-    :host(:not([aria-selected="true"]):active) {
-        background: ${neutralFillHover};
-        color: ${neutralForegroundRest};
-    }
-
-    :host([disabled]) {
-        cursor: ${disabledCursor};
-        opacity: ${disabledOpacity};
-    }
-
-    :host([disabled]:hover) {
-        background-color: inherit;
-    }
-
-    .content {
-        grid-column-start: 2;
-        justify-self: start;
-        overflow: hidden;
-        text-overflow: ellipsis;
-    }
-
-    .start,
-    .end,
-    ::slotted(svg) {
-        display: flex;
-    }
-
-    ::slotted(svg) {
-        /* TODO: adaptive typography https://github.com/microsoft/fast/issues/2432 */
-        height: calc(${designUnit} * 4px);
-        width: calc(${designUnit} * 4px);
-    }
-
-    ::slotted([slot="end"]) {
-        margin-inline-start: 1ch;
-    }
-
-    ::slotted([slot="start"]) {
-        margin-inline-end: 1ch;
-    }
-`.withBehaviors(
+        ${display("inline-flex")} :host {
+            position: relative;
+            font-family: ${bodyFont};
+            background: ${neutralFillStealthRest};
+            border-radius: calc(${controlCornerRadius} * 1px);
+            border: calc(${strokeWidth} * 1px) solid transparent;
+            box-sizing: border-box;
+            color: ${neutralForegroundRest};
+            cursor: pointer;
+            flex: 0 0 auto;
+            fill: currentcolor;
+            font-size: ${typeRampBaseFontSize};
+            height: calc(${heightNumber} * 1px);
+            line-height: ${typeRampBaseLineHeight};
+            outline: none;
+            overflow: hidden;
+            align-items: center;
+            padding: 0 calc(((${designUnit} * 3) - 3) * 1px );
+            user-select: none;
+            white-space: nowrap;
+        }
+        :host(:not([disabled]):hover),
+        :host(:not([disabled]):not([aria-selected="true"]):hover) {
+            background: ${neutralFillStealthHover};
+            color: ${neutralForegroundHover};
+        }
+        :host(:not([disabled]):active),
+        :host(:not([disabled]):not([aria-selected='true']):active),
+        :host([aria-selected="true"]),
+        :host(:not([disabled])[aria-selected="true"]:hover),
+        :host(:not([disabled])[aria-selected="true"]:active) {
+            background: ${neutralFillStealthActive};
+            border-color: ${accentFillRest};
+            color: ${neutralForegroundActive};
+        }
+        :host(:${focusVisible}) {
+            border-color: ${focusStrokeOuter};
+        }
+        :host([disabled]) {
+            cursor: ${disabledCursor};
+            opacity: ${disabledOpacity};
+        }
+        .content {
+            grid-column-start: 2;
```

Comment:
The ```host``` of content is inline-flex, I am not sure how placing this item through ```grid-column-start``` can be helpful.

Reply:
removed